### PR TITLE
Add tests for some utility functions

### PR DESCRIPTION
**Pull request by Twaha-Rahman**
**Description:**

This PR adds some tests for the following utility functions ->

- [x] `format-codeblock.ts`
- [x] `formatter.ts`
- [x] `is-supported-by-prettier.ts`
- [x] `bot-online-time.ts`

### CODE REVIEW

File: src/tests/format-codeblock.test.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    ['JSON', '{ "FCC": "Rocks!", "number": 123 }\n'],
+    ['Markdown', sampleMarkdown],
+    ['YAML', sampleYaml]
+  ];
+
+  it.each(testCases)(
+    'Should format `%s` codeblock',
+    (languageName, codeblock) => {
+      expect(formatCodeBlock(languageName, codeblock)).toBe(
+        `\`\`\`${languageName}\n${codeblock}\`\`\``
```
Comment (bradtaniguchi) [inline src/tests/format-codeblock.test.ts]: couldn't this be handled with:
```suggestion
        '```' + languageName + '\n' + codeblock + '```'
```
Or would that actually mess it up? (I haven't tested it yet myself on my side)

**Comment by bradtaniguchi** [comment]
Awesome thanks, I'll review this on my side later today :)

**Review by bradtaniguchi — COMMENTED** [review]
Just 1 place where I think we can simplify some string creation logic, besides that looks pretty good :+1:

File: src/tests/format-codeblock.test.ts (hunk 1 above)
Comment (Twaha-Rahman) [reply src/tests/format-codeblock.test.ts]: Applied it!

**Review by bradtaniguchi — APPROVED** [review]
LGTM 👍

**Pull request merged**